File: src/main/java/xyz/upperlevel/hgame/input/TriggerInputActionPacket.java
package xyz.upperlevel.hgame.input;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;
import xyz.upperlevel.hgame.network.Packet;
import xyz.upperlevel.hgame.network.ProtocolId;

@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
@ProtocolId(-1)
public class TriggerInputActionPacket implements Packet {
    private int actorId;
    private int actionId;
}

File: src/main/java/xyz/upperlevel/hgame/input/TriggerInputActionPacket.kt
package xyz.upperlevel.hgame.input

import xyz.upperlevel.hgame.network.Packet
import xyz.upperlevel.hgame.network.ProtocolId

@ProtocolId(2)
data class TriggerInputActionPacket(var actorId: Int, var actionId: Int) : Packet
